app/programs/page-client.tsx

```diff
@@ -0,0 +1,11 @@
+'use client';
+
+import { VerifiedProgramsCard } from '@/app/features/verified-programs';
+
+export default function ProgramsPageClient() {
+    return (
+        <div className="container mt-4">
```

Comment:
I think to keep `container` is fine, but could use Tailwind for margins: `e-mt-4`